docker/Dockerfile

```diff
@@ -0,0 +1,122 @@
+FROM alpine:3.3
```

Comment:
Want to stick with alpine until we see issues? 

If not I don't mind changing ti